recipes/ipython-sql/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.3.6" %}
+{% set md5 = "d4feb00ac5806d7640b2545a43974766" %}
+
+package:
+  name: ipython-sql
+  version: {{ version }}
+
+source:
+  fn: ipython-sql-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipython-sql/ipython-sql-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
```

Comment:
Guessing this is not needed now, but have forgotten. Maybe a reminder would be nice please. 😁